api/src/main/java/org/hawkular/apm/api/model/events/SourceInfo.java

```diff
@@ -70,31 +94,31 @@ public void setId(String id) {
     }
 
     /**
-     * @return the sourceUri
+     * @return the fragmentUri
```

Comment:
Did source uri represent source of the communication? What does represent fragmentUri now?